infrastructure/cloud_run.py

```diff
@@ -0,0 +1,62 @@
+from pulumi_gcp import serviceaccount, cloudrun, pubsub, cloudscheduler
+import base64
+from pulumi import Config
+import topics
+
+config = Config()
+
+api_key = config.require_secret("ALPACA_API_KEY_ID")
+api_secret = config.require_secret("ALPACA_API_SECRET_KEY")
```

Comment:
Include an `alpaca_*` prefix on the variable names (nitpick).

Reply:
oh, yea, good call, `api_key` is super generic


fixed

Reply:
fixed